src/main/java/pl/javastart/task/services/Contract.java

```diff
@@ -0,0 +1,54 @@
+package pl.javastart.task.services;
+
+abstract class Contract {
+    private int smsAmount;
+    private int mmsAmount;
+    private int voiceEventsTime;
+    private double creditBalance;
+
+    public int getSmsAmount() {
+        return smsAmount;
+    }
+
+    protected void setSmsAmount(int smsAmount) {
+        this.smsAmount = smsAmount;
+    }
+
+    public int getMmsAmount() {
+        return mmsAmount;
+    }
+
+    protected void setMmsAmount(int mmsAmount) {
+        this.mmsAmount = mmsAmount;
+    }
+
+    public int getVoiceEventsTime() {
+        return voiceEventsTime;
+    }
+
+    protected void setVoiceEventsTime(int voiceEventsTime) {
+        this.voiceEventsTime = voiceEventsTime;
+    }
+
+    public double getCreditBalance() {
+        return creditBalance;
+    }
+
+    protected void setCreditBalance(double creditBalance) {
+        this.creditBalance = creditBalance;
+    }
+
+    protected abstract boolean checkSmsAvailability();
+
+    protected abstract boolean checkCallAvailability(int seconds);
```

Comment:
Ta metoda powinna zwracać informacje ile sekund rozmowy udało się przeprowadzić. Jeśli na koncie mam środki na 1 minutę rozmowy, a chcę rozmawiać przez 2 minuty, to powinno zwrócić 60 sekund, a nie `false`.